src/content/app/tools/blast/views/blast-submission-results/components/job-parameters/JobParameters.tsx

```diff
@@ -50,6 +51,10 @@ const JobParameters = (props: JobParametersProps) => {
               <td>Database</td>
               <td>{databaseLabelsMap[parameters.database as DatabaseType]}</td>
             </tr>
+            <tr>
+              <td>Sensitivity</td>
+              <td>{props.preset}</td>
```

Comment:
![image](https://user-images.githubusercontent.com/6834224/192634873-ac569c8a-765b-47ad-84d3-421b3b10c46f.png)

you probably want this displayed differently 😃